src/genie/libs/parser/dell/tests/ShowIPRoute/cli/equal/golden_output1_output.txt

```diff
@@ -0,0 +1,2 @@
+S      *0.0.0.0/0 [254/1] via 10.10.21.1,   02d:06h:00m,  Vl20
```

Comment:
do you have any output with multiple nexthops like below? schema needs to be created after considering that case. 
```
S      *0.0.0.0/0 [254/1] via 10.10.21.1,   02d:06h:00m,  Vl20
                                        via 10.10.21.2,   02d:06h:00m,  Vl21
```
and how about `vrf`? you might want to have like:
```
'vrf': {
    'default': {
        'routes': {
(snip)